packages/office-ui-fabric-react/scripts/generateDefaultThemeSassFiles.js

```diff
@@ -0,0 +1,43 @@
+const { getTheme } = require('@uifabric/styling');
+const fs = require('fs');
+const path = require('path');
+
+const defaultTheme = getTheme(true);
+const fonts = defaultTheme.fonts;
+
+const lines = [];
+for (const fontName in fonts) {
+  const font = fonts[fontName];
+  for (const propName in font) {
+    const name = 'ms-font-' + fontName + '-' + propName;
+    lines.push(`$${name}: "[theme:${name}, default: ${font[propName]}]";`);
+  }
+}
+
+const srcRoot = './src/common';
+const fontsOutputFilename = '_themeVariables.scss';
+fs.writeFileSync(path.join(srcRoot, fontsOutputFilename), lines.join('\n'));
+
+// load palette
+
+const palette = defaultTheme.palette;
+const colorLines = [];
+for (const color in palette) {
+  const name = 'ms-font-' + color;
+  colorLines.push(`$${name}: "[theme:${color}, default: ${palette[color]}]";`);
+}
+
+const paletteOutputFilename = '_themeOverrides.scss';
+fs.writeFileSync(path.join(srcRoot, paletteOutputFilename), colorLines.join('\n'));
+
+// load semantic colors
+
+const semanticColors = defaultTheme.semanticColors;
+const semanticLines = [];
+for (const color in semanticColors) {
+  const name = color + 'Color';
+  semanticLines.push(`$${name}: '[theme:${color}, default: ${semanticColors[color]}]';`);
```

Comment:
i'm not totally sure how the deprecated comments are getting into the values below. Did you manually add them?

Ideally if someone else runs the tool again, it generates the same file.

Reply:
Weird, I'm not sure either! I'm not manually adding them. I'll investigate.